apps/editors/tests/test_views.py

```diff
@@ -2942,9 +2943,13 @@ class TestLimitedReviewerQueue(QueueTest, LimitedReviewerBase):
 
     def setUp(self):
         super(TestLimitedReviewerQueue, self).setUp()
-        self.expected_names = ['Nominated old']
         self.url = reverse('editors.queue_nominated')
 
+        for addon in self.generate_files().values():
```

Comment:
just FYI, this will give you a randomly ordered list. For this test it's ok but most of time you should rely on a predictably ordered list to guarantee that the test will pass all the time.